talk/objectorientation/objectsclasses.tex

```diff
@@ -49,16 +52,46 @@
       // let's square a
       myObj.squareA();
     \end{cppcode*}
-    \columnbreak
+      \onslide<2>
+    \begin{cppcode*}{gobble=2}
+      class MyFirstClass {
+        int a;
+        public:
+        void squareA() {
+          a *= a;
+        }
+        int sum(int b) {
+          return a + b;
+        }
+      };
+
+      MyFirstClass myObj;
+      myObj.a = 2; // error
+
+      // let's square a
+      myObj.squareA();
+    \end{cppcode*}
+    \end{overprint}
+      \columnbreak
```

Comment:
This will come later, when it's useful. I do not see the added value to repeat it here. It will only take more time to present and blur the message

Reply:
I will remove this now.